docs/contributor/03-30-trial-expiration.md

```diff
@@ -0,0 +1,15 @@
+# Trial expiration
+
+SAP BTP, Kyma runtime with Trial plan has a limited lifespan of 14 days counting from its creation time (as described in [Service description](../user/03-10-service-description.md#trial-plan)). After 14 days, [Trial Cleanup CronJob](./06-40-trial-cleanup-cronjob.md) sends a request to Kyma Environment Broker (KEB) to expire the trial instance. KEB suspends the instance without the ability to unsuspend it.
```

Comment:
In the sentence 'KEB suspends the instance without the ability to unsuspend it.', I'd use 'possibility' or 'option' instead of 'ability'.

Reply:
replaced `ability` with `possibility`